src/ansys/aedt/core/modules/material_lib.py

```diff
@@ -770,7 +770,7 @@ def find_datasets(d, out_list):
         return write_configuration_file(json_dict, output_file)
 
     @pyaedt_function_handler(full_path="input_file")
-    def import_materials_from_file(self, input_file=None, **kwargs):
+    def import_materials_from_file(self, input_file=None, stop_on_error: bool = True, **kwargs):
```

Comment:
it seems that stop_on_error is added but never used

Reply:
Good catch, initially I thought to add a flag to allow the user to say "I know what I'm doing" and continue/complete the import even if one or more materials failed to import.  I'll remove this.